cell/config.go

```diff
@@ -149,8 +149,15 @@ func (c *config[Cfg]) Apply(cont container, _ rootContainer) error {
 
 	// Register the flags to the global set of all flags.
 	err := cont.Invoke(
-		func(allFlags *pflag.FlagSet) {
+		func(allFlags *pflag.FlagSet) error {
+			var err error
+			flags.VisitAll(func(flag *pflag.Flag) {
+				if allFlags.Lookup(flag.Name) != nil {
+					err = fmt.Errorf("Duplicate flag %s", flag.Name)
```

Comment:
nit: could [join](https://pkg.go.dev/errors#Join) the errors, so that multiple duplicate flags would all be in the error message

```suggestion
					err = errors.Join(err, fmt.Errorf("Duplicate flag %s", flag.Name))
```